blog/docs/events/knative-at-kubecon-na-2022.md

```diff
@@ -0,0 +1,49 @@
+# Knative at KubeCon ’22 North America
+
+**Date: 2022-10-20**
+
+There's less than a week until
+[KubeCon + KnativeCon Europe 202](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/)
+kicks off in Detroit, MI.
+We are excited that the Knative community will be well represented at both events,
+and that, once again we are hosting a full day of Knative content at the
```

Comment:
```suggestion
and that once again we are hosting a full day of Knative content at the
```

Reply:
or `and that, once again, we are...`, but I think the first one's a little cleaner